### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by Ulberg** [owid/owid-grapher#2517] [Q&A]
Hey 👋🏾 

I have been wanting to contribute to your project, but from reading the CONTRIBUTION.MD it seemed like you preferred having some agency over what issues are done by externals. 

If you have any suggestions for issues that would be helpful for you to outsource i'd be more than happy to help.

**Comment by danyx23** [comment]
Hi @Ulberg! Sorry for the late reply, we are away in turn for vacation etc and thus the rest of the team is quite busy :).  

Can you say a few more words about what you'd be able to help with (i.e. what is your background/proficiency level), what you'd be interested in helping with and if you'd prefer to contribute small things once in a while or would like to work on a more substantial task eventually? Thanks!

**Reply by Ulberg** [reply]
Hey man, no worries 😊

I have a masters in AI and a bachelor in comp sci / software engineering. So i am fairly proficient with anything related to data science, and for work I have been a fullstack software engineer with an emphasis on frontend for some years. 

I prefer tasks related to frontend, but i´m generally cool with working on anything. 

If we could start with some smaller tasks, just to get familiar and then look into larger tasks after a while that would be perfect.

**Reply by Ulberg** [reply]
Speaking of vacations - i am just about to leave for a week now - so i won´t be able to follow up on this before next sunday

**Comment by Ulberg** [comment]
@danyx23 Hey, trying again, is there any way i can contribute to the software 😇
